perf/perf_uprobe.py

```diff
@@ -72,10 +73,10 @@ def cmd_verify(self, cmd):
 
     def test_uprobe(self):
         output = self.cmd_verify('%s /usr/bin/perf main' % self.cmdProbe)
-        if 'Added new event' not in output.stderr:
+        if 'Added new event' not in output.stderr.decode("utf-8"):
```

Comment:
why decode is required  i don't think it require if test run against avcoado 69 lts

Reply:
Yeah no need i will do changes